code/src/App.js

```diff
@@ -1,9 +1,73 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
+import Form from './components/Form'
+import ThoughtList from './components/ThoughtList'
+import './index.css'
+
+import { URL, LIKES_URL } from './reusable/url'
 
 export const App = () => {
+  const [messageList, setMessageList] = useState([])
+  const [messageNew, setMessageNew] = useState('')
+
+  useEffect(() => {
+    fetchMessageList()
+  }, [])
+
+  const fetchMessageList = () => {
+    fetch(URL)
+      .then((res) => res.json())
+      .then((messages) => setMessageList(messages))
+      .catch((err) => console.error(err))
+      setMessageNew('')
+  }
+
+  const handleMessageNewChange = (event) => {
+    setMessageNew(event.target.value)
+  }
+
+  const handleFormSubmit = (event) => {
+    event.preventDefault()
+
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ message: messageNew }),
```

Comment:
Very nice the space after "{" and before "}". Now I know why Maks mentioned that it would helps to read it easily - It does help! 🤓  🙃